stack.yaml

```diff
@@ -50,8 +50,8 @@ nix:
 allow-newer: true
 
 extra-deps:
-- servant-swagger-1.1.7.1@rev:2
-- swagger2-2.5
+# - servant-swagger-1.1.7.1@rev:2
+# - swagger2-2.5
```

Comment:
what happened with swagger?

Reply:
revision two was added to `14.27` https://www.stackage.org/lts-14.27/package/servant-swagger-1.1.7.1

`swagger2-2.5` I accidentally dropped. Good catch. However it still compiles. What was released in `2.5` that we need? 

I'll just add it back for now

Reply:
@fisx was there a specific reason why we were on 2.5 ?   Dropping it to 2.4 worked fine (no dependency conflicts or compiler errors)

Reply:
The `swagger2-2.5` release on hackage does not even compile for me.  I'm confused.  Given @fisx is maintainer of this package maybe he knows what is up here? https://github.com/wireapp/wire-server/pull/1030/commits/86ed4053cce9038eeb8fdca450ca889ace49f209

Reply:
[swagger2-2.5 is nice, but not important](https://github.com/GetShopTV/swagger2/releases/tag/v2.5).  I'll see if i can fix the issue upstream, then we can just wait for the next release.